fv3core/fv3core/_config.py

```diff
@@ -390,7 +390,13 @@ class Namelist:
     # use_ncep_sst: bool
     # blocksize: int
     # chksum_debug: bool
-    # dycore_only: bool
+    """
+    WARNING: dycore_only may not be used in this model
+     the same way it is in the Fortran version, watch for
+     consequences of these inconsistencies, or more closely
+     parallel the Fortran structure
```

Comment:
Nit: indentation, and allcaps warning is a little more alarm than I think is necessary for this. You could switch to a lowercase "warning" if you think "note" is not enough alarm.
```suggestion
    note: dycore_only may not be used in this model
       the same way it is in the Fortran version, watch for
       consequences of these inconsistencies, or more closely
       parallel the Fortran structure
```